packages/gasket-data/EXAMPLES.md

```diff
@@ -0,0 +1,126 @@
+# @gasket/data Examples
+
+This document provides working examples for all exported functions from `@gasket/data`.
+
+## gasketData()
```

Comment:
Using this in React components is not a great pattern to replicate. The `useGasketData` React hook is a better fit.

To show this in the browser, perhaps use it in a fetch wrapper and indicate that it must be called in the browser.

```js
import { gasketData } from '@gasket/data';

async function fetchSomeData() {
  if(typeof window !== 'undefined') {
    const data = gasketData();
    return fetch(data.someApiUrl)
  }
}
```

If you want to still show it in a React component, save it for an App Router example and give the example a `'use client'` directive.